### PR TITLE
Print function migration for PhysicsTools_PatAlgos

### PR DESCRIPTION
**Pull request by davidlange6**
Using futurize -n -w --no-diffs -j 4 -f libfuturize.fixes.fix_print_with_import

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by fabiocos** [comment]
+1

File: PhysicsTools/PatAlgos/python/tools/cmsswVersionTools.py [hunk 1]
```diff
@@ -203,40 +204,40 @@ def apply( self ):
 
         # Debugging output
         if debug:
-            print '%s DEBUG: Called with...'%( self._label )
+            print('%s DEBUG: Called with...'%( self._label ))
             for key in self._parameters.keys():
-               print '    %s:\t'%( key ),
-               print self._parameters[ key ].value,
+               print('    %s:\t'%( key ), end=' ')
+               print(self._parameters[ key ].value, end=' ')
```
Comment (slava77) [inline PhysicsTools/PatAlgos/python/tools/cmsswVersionTools.py]: is this ```, end=' '``` needed here?

Comment (slava77) [reply PhysicsTools/PatAlgos/python/tools/cmsswVersionTools.py]: answer to self: yes, it's needed.
```print ' ',``` doesn't end with newline and its replacement should have ```end=' '``` to replace the default ```\n```

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/24118	391f257
- technical
- jenkins tests pass

**Comment by fabiocos** [comment]
merge

**Pull request merged**